filters/filters.go

```diff
@@ -124,7 +124,7 @@ type FilterContext interface {
 	// Performs a new route lookup and executes the matched route if any
 	Loopback()
 
-	Logger() FilterContextLogger
+	Logger(...FilterContextLogger) FilterContextLogger
```

Comment:
This is not the best API design, it is not clear what happens if someone passes 0, 1, or many loggers.

Reply:
Okay, found some way not to change this API.